modules/security-network-egress.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * security/container_security/security-network.adoc
+
+[id="security-network-egress_{context}"]
+=  Egress traffic
+
+{product-title} provides the ability to control egress traffic using either
+a router or firewall method. For example, you can use IP whitelisting to control
```

Comment:
Suggest writing around "whitelisting"

Reply:
I'm going to leave it. The term "IP whitelisting" is well understood.

Reply:
Yeah that's fine. It's just one of the words we are going to look at replacing with master/slave, etc. so I was trying to get ahead of it. But we'll tackle it whenever we end up making those changes.